package.json

```diff
@@ -2,11 +2,14 @@
   "private": true,
   "scripts": {
     "build": "vitruvius build --src src --dest lib --ignore-packages \"**/integration-tests/**\"",
-    "clean": "rimraf ./packages/*/lib",
+    "prebuild": "rimraf ./packages/*/lib",
     "lint": "eslint . --cache",
-    "postinstall": "yarn run clean --silent && yarn run build --silent",
+    "postinstall": "yarn build --silent",
     "precommit": "lint-staged",
-    "test": "jest",
+    "setup": "lerna clean --yes && lerna bootstrap",
```

Comment:
Would you mind deleting your node modules folder and running yarn install again now? Since you're on 0.27.5 it ought to work without lerna bootstrapping.